application/views/admin/survey/subview/surveydashboard/004-generalsettingspanel.twig

```diff
@@ -41,7 +41,7 @@
                         {% if checkPermission('templates','read') %}
                             {% set sTemplateOptionsUrl = createUrl("themeOptions/updateSurvey", {'surveyid' : oSurvey.sid} ) %}
                             {% set sTemplateEditorUrl = createUrl("admin/themes/sa/view", {'templatename' : oSurvey.aOptions['template'] } ) %}
-                            {{templateModel.title}} ( {{templateModel.name}})
+                            {{templateModel.title}} ( {{templateModel.name}} )
```

Comment:
i didnt see any difference here, what was the goal?

Reply:
Right bracket didn't have space before it.

Reply:
Fruity Theme ( fruity)

Reply:
aha i thought the issue was aosomething different :) with the positionting of the button at the end not being eligned